lib/skate/settings/user.ex

```diff
@@ -75,7 +75,7 @@ defmodule Skate.Settings.User do
     Skate.Repo.all(from(u in users_for_route_ids_query(route_ids), select: u.id))
   end
 
-  def is_in_test_group(user_id, test_group_name) do
+  def in_test_group?(user_id, test_group_name) do
```

Comment:
This one is also dead code (it's only invoked in tests).

That's true as of removing the old search maps code, because I know we were using this to surface some data conditional on a user being in one or the other test group.

I'm on the fence about keeping it though, because it's not exactly dead code - just sleeping - and we will probably wake it up again during detour drawing. We could also use the `all_test_group_names` function to get the same information, and it would be better to only have this logic in one place.

Reply:
[Followup PR](https://github.com/mbta/skate/pull/2357)